backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -1771,6 +1786,77 @@ public static Order getOrderBasedOnSortKey(
     return finalPredicatesList;
   }
 
+  private static void setFuzzyOwnerPredicateBasedOnUACWorkspace(
+      CriteriaBuilder builder,
+      Root<?> entityRootPath,
+      AuthService authService,
+      ModelDBServiceResourceTypes modelDBServiceResourceTypes,
+      List<Predicate> keyValuePredicates,
+      KeyValueQuery predicate,
+      Operator operator,
+      RoleService roleService) {
+    if ((operator.equals(Operator.CONTAIN) || operator.equals(Operator.NOT_CONTAIN))) {
+      Predicate fuzzySearchPredicate =
+          getFuzzyUsersQueryPredicate(
+              authService,
+              roleService,
+              builder,
+              entityRootPath,
+              predicate,
+              modelDBServiceResourceTypes);
+      if (fuzzySearchPredicate != null) {
+        keyValuePredicates.add(fuzzySearchPredicate);
+      } else {
+        throw new ModelDBException(
+            ModelDBConstants.INTERNAL_MSG_USERS_NOT_FOUND, Code.FAILED_PRECONDITION);
+      }
+    } else {
+      String ownerId = predicate.getValue().getStringValue();
+      Map<String, UserInfo> userInfoMap =
+          authService.getUserInfoFromAuthServer(
+              new HashSet<>(Collections.singleton(ownerId)),
+              Collections.emptySet(),
+              Collections.emptyList());
+      Set<String> resourceIdSet =
+          getResourceIdsFromUserWorkspaces(
+              authService,
+              roleService,
+              modelDBServiceResourceTypes,
+              Collections.singletonList(userInfoMap.get(ownerId)));
+      Expression<String> exp = entityRootPath.get(ModelDBConstants.ID);
+      keyValuePredicates.add(exp.in(resourceIdSet));
+    }
+  }
+
+  private static void setFuzzySearchPredicateBasedOnMDBOwner(
```

Comment:
https://github.com/VertaAI/modeldb/pull/1838/files#r554756537
This method used for other entities excluding project and repository which still used the owner column at the MDB.